strax/processing/peak_splitting.py

```diff
@@ -189,6 +170,54 @@ def _split_peaks(split_finder, peaks, orig_dt, is_split, min_area,
 
         yield offset
 
+    @staticmethod
+    @strax.growing_result(dtype=strax.hitlet_dtype(), chunk_size=int(1e4))
+    @numba.jit(nopython=True, nogil=True)
+    def _split_hitlets(split_finder, hits, orig_dt, is_split, min_area,
+                       args_options,
+                       _result_buffer=None, result_dtype=None):
+        """Loop over peaks, pass waveforms to algorithm, construct
+        new peaks if and where a split occurs.
+        """
+        new_hits = _result_buffer
+        offset = 0
+
+        for p_i, p in enumerate(hits):
+            if p['area'] < min_area:
+                continue
+
+            prev_split_i = 0
+            w = p['data'][:p['length']]
+            for split_i, bonus_output in split_finder(
+                    w, p['dt'], p_i, *args_options):
+                if split_i == NO_MORE_SPLITS:
+                    return
+                r['record_i'] = p['record_i']
+
+                is_split[p_i] = True
+                r = new_hits[offset]
+                r['time'] = p['time'] + prev_split_i * p['dt']
+                r['channel'] = p['channel']
+                # Set the dt to the original (lowest) dt first;
+                # this may change when the sum waveform of the new peak
+                # is computed
+                r['dt'] = orig_dt
+                r['length'] = (split_i - prev_split_i) * p['dt'] / orig_dt
+                r['max_gap'] = -1  # Too lazy to compute this
```

Comment:
hitlets does not support 'max_gap' so simply remove this line.

Reply:
Fixed in 17b4c4e